Documentation/Examples/MSBuild/DeterministicBuild/Directory.Build.targets

```diff
@@ -1,4 +1,4 @@
 <Project>
   <!-- Deterministic build workaround -->
-  <Import Project="$(MSBuildThisFileDirectory)DeterministicBuild.targets" />
+  <!--<Import Project="$(MSBuildThisFileDirectory)DeterministicBuild.targets" />-->
```

Comment:
should we completely remove or do we need an issue here too to not forget?

Reply:
This should have beed removed since the **net6.0 migration** and this should be removed. I did not remove it beacuse we have also the target file in the repository root [DeterministicBuild.targets](https://github.com/coverlet-coverage/coverlet/blob/master/DeterministicBuild.targets)